packages/website/src/common/Search/index.tsx

```diff
@@ -13,18 +13,34 @@ import Autocomplete from "@material-ui/lab/Autocomplete";
 import { setReefOnMap } from "../../store/Homepage/homepageSlice";
 import type { Reef } from "../../store/Reefs/types";
 import { reefsListSelector } from "../../store/Reefs/reefsListSlice";
+import { formatReefName } from "../../store/Reefs/helpers";
 
 const Search = ({ classes }: SearchProps) => {
   const [searchedReef, setSearchedReef] = useState<Reef | null>(null);
   const dispatch = useDispatch();
-  const reefs = useSelector(reefsListSelector).filter(({ name }) => name);
+  const reefs = useSelector(reefsListSelector)
+    .filter((reef) => formatReefName(reef))
```

Comment:
```suggestion
    .filter(formatReefName)
```
I think you can just pass the function reference here if you wish